plugins/flutter_aepedgeidentity/android/gradle.properties

```diff
@@ -0,0 +1,3 @@
+org.gradle.jvmargs=-Xmx1536M
+android.useAndroidX=true
+android.enableJetifier=true
```

Comment:
the edge identity extension doesn't seem to use android dependencies, why is jetifier required here?

Reply:
Do I need to remove android.enableJetifier=true for Consent and Edge?

Reply:
those two don't use platform specific APIs either